spec/features/10.admin_signs_in_spec.rb

```diff
@@ -0,0 +1,69 @@
+feature 'Admin signs into website' do
+  context 'As an Admin' do
+    before do
+      admin = User.create(name: 'admin', email: 'specialemail@company.com', password: 'supersecret', role: 'admin' )
+      login_as(admin, scope: :user, run_callbacks: false)
+    end
+
+    scenario 'I can sign in and see a special message on the homepage' do
+      devtools = FactoryGirl.create_list(:devtool, 5 )
+      visit root_path
+      expect(page).to have_content( %Q{
+        Welcome Super Elite Group 8 member,
+        you are a member of an exclusive, Clandestineand,
+        and Dope AF club.} )
+      devtools.each do |devtool|
+        expect(page).to have_content('delete')
+      end
+    end
```

Comment:
you dont need to iterate over every one, simply make sure one devtool has the delete